draft-ietf-quic-recovery.md

```diff
@@ -1151,12 +1157,13 @@ Pseudocode for OnLossDetectionTimeout follows:
 
 ~~~
 OnLossDetectionTimeout():
-  loss_time, pn_space = GetEarliestLossTime()
-  if (loss_time != 0):
-    // Time threshold loss Detection
-    DetectLostPackets(pn_space)
-    SetLossDetectionTimer()
-    return
+  if (timer_mode == LOSS_DETECTION):
+    loss_time, pn_space = GetEarliestLossTime()
+    if (loss_time != 0):
```

Comment:
So, this should not be required. Specifically, I think what we might want here is an ASSERT(loss_time != 0), though an assert might be too harsh. Maybe a warning and return.

Also below, under the else for a PTO, add an assert that mode is PTO.

Reply:
I think we either need an if check or an ASSERT.  The following code does not work correctly if you did enter it code and loss_time was 0.